relay-server/src/metrics_extraction/spans/snapshots/relay_server__metrics_extraction__spans__tests__extract_span_metrics.snap

```diff
@@ -320,13 +320,13 @@ expression: event.value().unwrap().spans
                 "http",
             ),
             "span.description": String(
-                "POST *.domain.tld:targetport/<unparameterized>",
+                "POST http://*/api/hi",
```

Comment:
This is a side effect of changing the test input: `"targetport"` is not a valid port parsable by `Url`, so I replaced it with 1234. Now, the entire port + domain get scrubbed by `scrub_identifiers`. I will follow up on this in a later PR.